### ISSUE TITLE
getLastChild function should ignore comment node

### ISSUE THREAD

**Issue by yangxuanxing** [CezaryDanielNowak/React-dotdotdot#34]
clamp.js line: 130

function getLastChild should ignore comment node

**Comment by CezaryDanielNowak** [comment]
Agree. Besides that I opt for having configurable list of tags to omit: `a`, `span`, `button`, `strong`, `abbr`, `sub`, `sup` should be there by default.

**Comment by CezaryDanielNowak** [comment]
Inserting comments with react  is a bit tricky, but I think I managed to fix your issue @yangxuanxing. Please let me know if it's fixed in 1.3.0
